seungjong/app.js

```diff
@@ -112,6 +112,68 @@ app.get('/users/:userId/posts', async (req, res, next) => {
 });
 
 
+app.patch('/users/:userId/:postId', async (req, res, next) => {
+    const { userId, postId } = req.params;
+    const { userName, title, content } = req.body;
+
+    await dataSource.query(`    
+        UPDATE posts
+        SET
+        content = ?
+        WHERE id = ?     
+        `, [content, postId]
+        )
+    
+    const result = await dataSource.query(`
+        SELECT
+        u.id,
+        u.name,
+        p.id,
+        p.title,
+        p.content
+        FROM users as u
+        JOIN posts as p
+        ON u.id = p.user_id
+        WHERE p.id = ?;
+    `, [postId]);
+
+    res.status(200).json({ data: {
+        userId : userId,
+        userName: result[0].name,
+        postingId : postId,
+        postingTitle: result[0].title,
+        postingContent: content
+    } });
+})
+
+
+app.delete('/users/:userId/:postId/postDelete', async(req, res, next) => {
```

Comment:
```suggestion
app.delete('/users/:userId/posts/:postId', async(req, res, next) => {
```
1. 엔드포인트에 동작이 들어가지 않도록 해주세요. 
2. 엔드포인트 작성 순서를 바꿔주셔야 의도파악이 자연스럽습니다.